pom.xml

```diff
@@ -68,7 +68,7 @@
     <jackson.datatype.groupId>com.fasterxml.jackson.datatype</jackson.datatype.groupId>
     <jackson.package>com.fasterxml.jackson</jackson.package>
     <jackson.version>2.17.1</jackson.version>
-    <jackson-databind.version>2.17.1</jackson-databind.version>
+    <jackson-databind.version>2.17.2</jackson-databind.version>
```

Comment:
@Fokko Should we upgrade these jackson dependencies all together?

Reply:
@wgtmac Ideally yes, it should go together with https://github.com/apache/parquet-java/pull/2964. In the past, there were sometimes minor updates of one of the libraries. For example `2.17.2.1` for `jackson-databind`. Therefore there are separate properties for `jackson` and `jackson-databind`.

Reply:
Make sense. Thanks!